src/api/parser/src/utils/__mocks__/supabase.js

```diff
@@ -0,0 +1,73 @@
+const { hash } = require('@senecacdot/satellite');
+const normalizeUrl = require('normalize-url');
+
+const urlToId = (url) => hash(normalizeUrl(url));
+
+let feeds = [];
+let feedIds = new Set();
+
+module.exports = {
+  __resetMockFeeds: () => {
+    feeds = [];
+    feedIds = new Set();
+  },
+  /**
+   * @param {Array<Feed | { url: string }>} feedObjects
+   */
+  __setMockFeeds: (feedObjects) => {
+    const mockFeeds = feedObjects.reduce((uniqueFeeds, feed) => {
+      const id = feed.id || urlToId(feed.url);
+      if (!feedIds.has(id)) {
+        feedIds.add(id);
+        return uniqueFeeds.concat(feed);
```

Comment:
Think you need to add a 
```
feed.id = id;
```
Somewhere in this if statement, before the return

Reply:
Oops, I forgot about this, thanks